src/event/ngx_event_openssl.h

```diff
@@ -224,6 +224,11 @@ ngx_int_t ngx_ssl_ocsp_get_status(ngx_connection_t *c, const char **s);
 void ngx_ssl_ocsp_cleanup(ngx_connection_t *c);
 ngx_int_t ngx_ssl_ocsp_cache_init(ngx_shm_zone_t *shm_zone, void *data);
 
+void *ngx_ssl_cache_fetch(ngx_conf_t *cf, ngx_uint_t index, char **err,
+    ngx_str_t *path, void *data);
+void *ngx_ssl_cache_connection_fetch(ngx_pool_t *pool, ngx_uint_t index,
```

Comment:
The name is somewhat misleading. I understand where it came from though. An alternative could be `ngx_ssl_cache_fetch_nocache()` for example.

Reply:
Well, it can return cacheable objects in theory, that's why it doesn't inherit such naming.
Also it is used to be called in runtime, as opposite to config time (note the absence of ngx_conf_t argument).